src/main/resources/lang/strings_en_US.properties

```
@@ -101,7 +101,7 @@ MusicManagerTrackLoaded={1} has been added.
 MusicManagerPlaylistLoaded=Playlist {1} has been added :
 MusicManagerPlaylistAddedTrackInfo=Added track `#{1}` **->** {2}.
 MusicManagerNoMatch=The song {1} has not been found.
-MusicManagerLoadFailed=Cannot load the song : {1}.
+MusicManagerLoadFailed=Cannot load track. Reason  {1}
```

Should we have a ":", like so :
```suggestion
MusicManagerLoadFailed=Cannot load track. Reason : {1}
```

?

---

Yeah